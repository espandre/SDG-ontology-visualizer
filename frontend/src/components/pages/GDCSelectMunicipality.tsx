import { Heading, Stack, Flex } from '@chakra-ui/react';
import React from 'react';

import MunicipalityList from '../molecules/MunicipalityList';

const SelectMunicipality: React.FC = () => (
  <Stack spacing="10">
    <Flex
      align="center"
      justify="center"
      justifyContent="space-evenly"
      h="150px"
      spacing="10"
      bg="cyan.700"
    >
      <Stack spacing="10">
        <Heading size="lg" color="white">
          Select municipality
        </Heading>
      </Stack>
    </Flex>
    <MunicipalityList />
  </Stack>
);

export default SelectMunicipality;
