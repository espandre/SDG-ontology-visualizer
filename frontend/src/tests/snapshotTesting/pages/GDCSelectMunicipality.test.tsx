import React from 'react';
import renderer from 'react-test-renderer';

import GDCSelectMunicipality from '../../../components/pages/GDCSelectMunicipality';

it('renders when there are no items', () => {
  const tree = renderer.create(<GDCSelectMunicipality />).toJSON();
  expect(tree).toMatchSnapshot();
});
