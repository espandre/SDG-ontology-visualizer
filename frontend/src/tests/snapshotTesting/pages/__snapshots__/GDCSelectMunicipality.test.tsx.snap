// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no items 1`] = `
<div
  className="chakra-stack css-efzghl"
>
  <div
    className="css-ixqi0y"
    spacing="10"
  >
    <div
      className="chakra-stack css-efzghl"
    >
      <h2
        className="chakra-heading css-kbn7if"
      >
        Select municipality
      </h2>
    </div>
  </div>
  <div
    className="css-172bykr"
  >
    <div
      className="chakra-stack css-eao00i"
    >
      <div
        className="chakra-spinner css-1nasj0w"
      >
        <span
          className="css-f8n5zr"
        >
          Loading...
        </span>
      </div>
      <p
        className="chakra-text css-0"
      >
        Loading...
      </p>
    </div>
  </div>
</div>
`;
